packages/remix/README.md

```diff
@@ -71,24 +71,22 @@ import {
   ScrollRestoration,
 } from "@remix-run/react";
 
-import { ErrorBoundary, withSentryRouteTracing } from "@sentry/remix";
+import { withSentryRouteTracing } from "@sentry/remix";
```

Comment:
We can keep `withSentryRouteTracing` export, deprecate it, and then export `withSentry`